packages/elements/src/Text/utils/styles/addMarginsToLists.ts

```diff
@@ -0,0 +1,29 @@
+const listMargins = (node: Element): void => {
+  const allowedTags = ["UL", "OL"];
+  if (allowedTags.includes(node.nodeName)) {
+    const { marginTop, marginBottom } = window.getComputedStyle(node);
+
+    if (marginTop) {
```

Comment:
the value of `marginTop` contain the `px` not an integer is a string, the `classNames` in brizy not contain the suffix `px`

- GetComputedStyle
![image](https://github.com/bagrinsergiu/MB-migration/assets/18303258/5e7b7cc2-5039-4a45-87aa-f37692a6dab2)



- QuillJS
![image](https://github.com/bagrinsergiu/MB-migration/assets/18303258/1c4781b7-cc7e-4c24-834b-c6b2795c79d3)